templates/components/form/inventory_info.html.twig

```diff
@@ -125,6 +123,7 @@
          });
 
          $('#update-inventory').on('click', function() {
+            var icon = $(this);
             icon.addClass("fa-spin");
             $.ajax({
                type: 'GET',
```

Comment:
It will trigger an indirect writing operation, so it should use POST method.
```suggestion
               type: 'POST',
```

Reply:
done